lib/rust/ensogl/core/src/control/io/mouse.rs

```diff
@@ -53,22 +59,35 @@ macro_rules! define_bindings {
         #[derive(Clone,CloneRef,Debug,Default)]
         #[allow(missing_docs)]
         pub struct MouseManagerDispatchers {
-            $(pub $name : callback::registry::Ref1<$target>),*
+            $(pub $name : callback::registry::Ref1<$event>),*
         }
 
         impl MouseManager {
-            /// Constructor.
+            /// Constructor. See docs of `new_separated` to learn more.
```

Comment:
```suggestion
            /// Constructor. See docs of [`Self::new_separated`] to learn more.
```
Or [`new_separated`](Self::new_separated).

You can check in IntelliJ if the doc links work by changing to rich mode, clicking on the links and check if proper docs are displayed.

Reply:
The idea with checking it in IntelliJ is amazing, but somehow it does not work on my end. Have seen it before?
<img width="510" alt="image" src="https://user-images.githubusercontent.com/1623053/226005262-d58b514c-8b13-4d65-b7a6-399380e3356c.png">